transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionExtension.java

```diff
@@ -163,11 +169,12 @@ private void registerTransformers(SubsystemRegistration subsystem) {
 
 
         // 2.0.0 --> 1.5.0
-        ResourceTransformationDescriptionBuilder builderEap64 = chainedBuilder.createBuilder(subsystem.getSubsystemVersion(), MODEL_VERSION_EAP64);
+        ResourceTransformationDescriptionBuilder builderEap64 = chainedBuilder.createBuilder(MODEL_VERSION_WFLY10, MODEL_VERSION_EAP64);
         builderEap64.getAttributeBuilder()
                 .addRename(TransactionSubsystemRootResourceDefinition.USE_JOURNAL_STORE, CommonAttributes.USE_HORNETQ_STORE)
                 .addRename(TransactionSubsystemRootResourceDefinition.JOURNAL_STORE_ENABLE_ASYNC_IO, CommonAttributes.HORNETQ_STORE_ENABLE_ASYNC_IO)
-                .addRename(TransactionSubsystemRootResourceDefinition.STATISTICS_ENABLED, CommonAttributes.ENABLE_STATISTICS);
+                .addRename(TransactionSubsystemRootResourceDefinition.STATISTICS_ENABLED, CommonAttributes.ENABLE_STATISTICS)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, TransactionSubsystemRootResourceDefinition.USE_ACTIONSTATUSSERVICE_RECOVERY_FILTER);
```

Comment:
This is a boolean which can have one of two values (and also undefined, which becomes the default value). It should be discarded if it has the value which is the behaviour the old hosts use, and rejected if the value it is the new behaviour not available/configurable on the old host.